packages/cli-platform-apple/src/tools/listDevices.ts

```diff
@@ -49,9 +49,45 @@ const parseXcdeviceList = (text: string, sdkNames: string[] = []): Device[] => {
   return devices;
 };
 
+/**
+ * Executes `xcrun xcdevice list` and `xcrun simctl list --json devices`, and connects parsed output of these two commands. We are running these two commands as they are necessary to display both physical devices and simulators. However, it's important to note that neither command provides a combined output of both.
+ * @param sdkNames
+ * @returns List of available devices and simulators.
+ */
 async function listDevices(sdkNames: string[]): Promise<Device[]> {
-  const out = execa.sync('xcrun', ['xcdevice', 'list']).stdout;
-  return parseXcdeviceList(out, sdkNames);
+  const xcdeviceOutput = execa.sync('xcrun', ['xcdevice', 'list']).stdout;
+  const parsedXcdeviceOutput = parseXcdeviceList(xcdeviceOutput, sdkNames);
+
+  const simctlOutput = JSON.parse(
+    execa.sync('xcrun', ['simctl', 'list', '--json', 'devices']).stdout,
+  );
```

Comment:
can you measure whether running these 2 commands in parallel is faster or not?

Reply:
So, I gave it a shot, and used `performance.now()` to measure 5 times execution time in parallel and in sequential run, here are the result:

| Parallel run | Sequential run |
|--------|--------|
| 5601.046166986227 | 5644.935749977827 | 
| 5649.5136660039425 | 5632.401457995176 | 
| 5634.3378329873085 | 5684.918291002512 | 
| 5606.314500004053 | 5615.477707982063 | 
| 5674.328749984503 | 5632.90391600132 | 
| AVG: **5633.108183** | AVG: **5642.127425** |

(Results in milliseconds.)

The difference is really slow, so I think we can leave it as it is 👍